Linux npm install protagonist -g fails.

Trying to get apiary with api-mock running on my machine. Keep getting install errors for protagonist:

```
> protagonist@1.2.6 install /usr/local/lib/node_modules/protagonist
> node-gyp rebuild

make: Entering directory `/usr/local/lib/node_modules/protagonist/build'
  CXX(target) Release/obj.target/libmarkdownparser/drafter/ext/snowcrash/ext/markdown-parser/src/ByteBuffer.o
make: g++: Command not found
make: *** [Release/obj.target/libmarkdownparser/drafter/ext/snowcrash/ext/markdown-parser/src/ByteBuffer.o] Error 127
make: Leaving directory `/usr/local/lib/node_modules/protagonist/build'
gyp ERR! build error 
gyp ERR! stack Error: `make` failed with exit code: 2
gyp ERR! stack     at ChildProcess.onExit (/usr/local/lib/node_modules/npm/node_modules/node-gyp/lib/build.js:276:23)
gyp ERR! stack     at emitTwo (events.js:100:13)
gyp ERR! stack     at ChildProcess.emit (events.js:185:7)
gyp ERR! stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:200:12)
gyp ERR! System Linux 3.19.0-32-generic
gyp ERR! command "/usr/local/bin/node" "/usr/local/lib/node_modules/npm/node_modules/node-gyp/bin/node-gyp.js" "rebuild"
gyp ERR! cwd /usr/local/lib/node_modules/protagonist
gyp ERR! node -v v5.6.0
gyp ERR! node-gyp -v v3.2.1
gyp ERR! not ok 
npm ERR! Linux 3.19.0-32-generic
npm ERR! argv "/usr/local/bin/node" "/usr/local/bin/npm" "install" "protagonist" "-g"
npm ERR! node v5.6.0
npm ERR! npm  v3.6.0
npm ERR! code ELIFECYCLE

npm ERR! protagonist@1.2.6 install: `node-gyp rebuild`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the protagonist@1.2.6 install script 'node-gyp rebuild'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the protagonist package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     node-gyp rebuild
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs protagonist
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!     npm owner ls protagonist
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:

```

OSX 10.10.5
node 5.3.0
npm 3.8.1

$ npm install -g protagonist

CXX(target) Release/obj.target/protagonist/src/parse_async.o
../src/parse_async.cc:134:15: warning: 'FatalException' is deprecated: Use
      FatalException(isolate, ...) [-Wdeprecated-declarations]
        node::FatalException(try_catch);
              ^
/Users/rafael/.node-gyp/5.3.0/include/node/node.h:283:29: note: 'FatalException' has
      been explicitly marked deprecated here
                inline void FatalException(const v8::TryCatch& try_catch) {
                            ^
/Users/rafael/.node-gyp/5.3.0/include/node/node.h:66:42: note: expanded from macro
      'NODE_DEPRECATED'
    __attribute__((deprecated(message))) declarator
                                         ^
1 warning generated.
  CXX(target) Release/obj.target/protagonist/src/parse_sync.o
  CXX(target) Release/obj.target/protagonist/src/protagonist.o
  CXX(target) Release/obj.target/protagonist/src/result.o
  CXX(target) Release/obj.target/protagonist/src/v8_wrapper.o
  SOLINK_MODULE(target) Release/protagonist.node
ld: library not found for -lgcc_s.10.5
clang: error: linker command failed with exit code 1 (use -v to see invocation)
make: *** [Release/protagonist.node] Error 1
gyp ERR! build error 
gyp ERR! stack Error: `make` failed with exit code: 2
gyp ERR! stack     at ChildProcess.onExit (/usr/local/lib/node_modules/npm/node_modules/node-gyp/lib/build.js:276:23)
gyp ERR! stack     at emitTwo (events.js:87:13)
gyp ERR! stack     at ChildProcess.emit (events.js:172:7)
gyp ERR! stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:200:12)
gyp ERR! System Darwin 14.5.0
gyp ERR! command "/usr/local/Cellar/node/5.3.0/bin/node" "/usr/local/lib/node_modules/npm/node_modules/node-gyp/bin/node-gyp.js" "rebuild"
gyp ERR! cwd /usr/local/lib/node_modules/protagonist
gyp ERR! node -v v5.3.0
gyp ERR! node-gyp -v v3.3.0
gyp ERR! not ok 
npm ERR! Darwin 14.5.0
npm ERR! argv "/usr/local/Cellar/node/5.3.0/bin/node" "/usr/local/bin/npm" "install" "-g" "protagonist"
npm ERR! node v5.3.0
npm ERR! npm  v3.8.1
npm ERR! code ELIFECYCLE

npm ERR! protagonist@1.2.6 install: `node-gyp rebuild`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the protagonist@1.2.6 install script 'node-gyp rebuild'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the protagonist package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     node-gyp rebuild
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs protagonist
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!     npm owner ls protagonist
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     /Users/rafael/Work/npm-debug.log

--- a/README.md
+++ b/README.md
@@ -17,7 +17,9 @@ The best way to install Protagonist is by using its [NPM package](https://npmjs.
 $ npm install protagonist
 ```
 
-Works smoothly with node.js ~0.8.15, 0.10.x, 0.12.x, 2 to 4 and iojs v1.x.x
+**NOTE:** *Installing Protagonist depends on having Python 2 installed along with a compiler such as GCC, Clang or MSVC.*
+
+Tested with node.js ~0.8.15, 0.10.x, 0.12.x, 2 to 5 and iojs v1.x.x
 
 ## Getting started
 

